ci/pipelines/cf-for-k8s-prs.yml

```diff
@@ -49,24 +49,22 @@ test-runner-config: &common-test-config
 jobs:
 - name: run-tests-on-cf-for-k8s-pr
   public: true
-  serial: true
   plan:
-  - timeout: 4h
```

Comment:
oh I guess this could still be useful after the pool lock is acquired. Maybe in case someone accidentally or maliciously make the deploy or tests hang somehow.